libraries/botbuilder-core/src/activityHandler.ts

```diff
@@ -3,12 +3,20 @@
  * Licensed under the MIT License.
  */
 
-import { MessageReaction, HealthCheckResponse } from 'botframework-schema';
+import {
+    Activity,
+    AdaptiveCardInvokeResponse,
+    AdaptiveCardInvokeValue,
+    HealthCheckResponse,
+    MessageReaction,
+} from 'botframework-schema';
+
 import { ActivityHandlerBase } from './activityHandlerBase';
+import { InvokeException } from './invokeException';
```

Comment:
@joshgummersall I think these change should be in ActivityHandlerBase which was created to make Invoke-handling clearer.

Reply:
A faithful port of https://github.com/microsoft/botbuilder-dotnet/pull/5227/files#diff-7a382816f3c128c2daeb75ea350f34c5410ff57febaa968ada7e17dabb039224 - are we intentionally doing something differently?

Reply:
ActivityHandlerBase is equivalent to ActivityHandler in .NET. By porting the changes to the subclass instead of the base class, customers inheriting from the base class won't pick up the changes.

Reply:
Ah - good deal, that makes sense 😄

Reply:
@stevengum we discussed briefly on Teams but I don't recall what the conclusion is here - do I move all this logic to the base class?